controllers/FollowController.js

```diff
@@ -43,8 +43,38 @@ const getFollowStatus = async (req, res) => {
   }
 };
 
+const getFollowersUserList = async (req, res) => {
+  try {
+    const { visitedUserId, currentUserId } = req.query;
```

Comment:
I don't think we need to user ids here, same with in `getFollowingUserList`. If we want to look for whoever's following user A, we only need user A's user id. Same for if we're looking for whoever user A's following

Reply:
I think we need two userIds, mainly because of the `isFollowing` status. If we are to get the list of users that follow an account or a list of users that the visited account follows, then we would need only one userId.

The `isFollowing` status is dependent on the currently logged in user crossed with the visited user's followers and following users, not with the the visited account.